tests/cephfs/cephfs_utilsV1.py

```diff
@@ -968,6 +981,93 @@ def get_all_subvolumes(self, client, fs_list):
             subvolumes.extend([i["name"] for i in all_sub_info])
         return subvolumes
 
+    @function_execution_time
+    def create_nfs(self, client, nfs_cluster_name, validate=True, **kwargs):
+        """
+
```

Comment:
Can we add description to what the function is meant to achieve along with data type of the args passed and return values.